GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -108,7 +114,11 @@ internal void HandleAssetBundleVerifyState(AssetBundleVerifyState state, UnityWe
                     _bundle.Unload(true);
                     break;
                 default:
-                    throw new NotImplementedException(string.Format("Unexpected state {0}", state));
+                    var errorMessage =
+                        string.Format(
+                            "An unexpected state was found while verifying the AssetBundle. Unexpected state: {0}",
```

Comment:
"Unexpected state while checking the AssetBundle: {0}"

Reply:
Resolved.